pallets/stateful-storage/src/lib.rs

```diff
@@ -73,17 +73,16 @@ pub use weights::*;
 #[frame_support::pallet]
 pub mod pallet {
 	use super::*;
-	use crate::{
-		stateful_child_tree::{StatefulChildTree, StatefulPageKeyPart},
-		types::{ItemAction, Page},
-	};
+	use crate::{stateful_child_tree::StatefulChildTree, types::*};
 	use common_primitives::{
 		msa::{MessageSourceId, MsaLookup, MsaValidator, SchemaGrantValidator},
 		schema::{SchemaId, SchemaProvider},
 		stateful_storage::PageId,
 	};
 	use frame_support::pallet_prelude::*;
 
+	type Tree = StatefulChildTree<Identity>;
+
```

Comment:
:1st_place_medal:

Reply:
There might be a miscommunication on my part. We want `TwoX` for this pallet but for tests and maybe other pallets the `Identity` default makes sense

Reply:
Right; I was waiting for a decision before updating this. I'll change to `TwoX`.